library/js/oeUI/headerTitleAction.js

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
Note you need to end this with .php for it go through php processing:
headerTitleAction.js.php

Reply:
This is likely dependent on server. For example, in a nginx/php-fpm setup, only scripts that contain either a .php or .inc ending will be run through a php compiler.